meson.build

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: GPL-2.0
+
+project('cdba', 'c')
+
+client_srcs = ['cdba.c',
```

Comment:
And you have lost all the compiler flags carefully picked up by @krzk

Reply:
I assume meson carefully select nice flags depending on compiler and source code language, but I can check if it’s necessary to add those flags

Reply:
OK so I added them and it fails on older releases...

https://github.com/linux-msm/cdba/actions/runs/6377506014/job/17306303374

It seems the Makefile doesn't add them for xenial & bionic ?

Reply:
Yep. See #34

Reply:
Solved but the meson version on xenial is so old we can't even test the meson version....